src/metadata.json

```diff
@@ -1 +1 @@
-{"shell-version": ["3.20.4"], "description": "Show logout button on the system menu", "uuid": "please_log_out@bacchus.snucse.org", "name": "Please log out"}
+{"shell-version": ["3.20.4". "3.22"], "description": "Show logout button on the system menu", "uuid": "please_log_out@bacchus.snucse.org", "name": "Please log out"}
```

Comment:
Thanks for your PR, but could you change `["3.20.4". "3.22"]` to `["3.20.4", "3.22"]`?